server/config/type.js

```diff
@@ -221,6 +236,25 @@ const ConfigType = class ConfigType {
         formattedConfig[relationName] = relations;
       }));
 
+      this.populate
+        .filter((populatedFields) => !populatedFields.includes("."))
```

Comment:
Can you tell me why you _explicitly exclude_ populate strings that contain a `.`?

Reply:
You can also use populate for multiple levels, for example:
![image](https://github.com/boazpoolman/strapi-plugin-config-sync/assets/103750528/e25bf1b0-6667-428f-bed7-3241d7fff434)

The levels are separated by ".". At this point, we do not need to clean up each levels, the main level is enough.

Reply:
Ah that makes sense. I guess that's something we would have to write in the README documentation as well.